Bottom: 5ea685cdd66a1eb2652ee9d6ead781c8cc4a6a5d
Top:    29bb35f0ad4c1a493cd13d714f5c76c4b35a2474
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-03-06 23:21:30 +1300

Refresh of placeholder-tags

---

diff --git a/testtools/testcase.py b/testtools/testcase.py
index 97158c0..52dfde9 100644
--- a/testtools/testcase.py
+++ b/testtools/testcase.py
@@ -634,7 +634,8 @@ class PlaceHolder(object):
         self._outcome = outcome
         if error is not None:
             self._details['traceback'] = content.TracebackContent(error, self)
-        self._tags = tags or set()
+        tags = tags or frozenset()
+        self._tags = frozenset(tags)
 
     def __call__(self, result=None):
         return self.run(result=result)
